src/commands/deploy/deployFromRegistry/dockerHub/DockerHubV2ApiCalls.ts

```diff
@@ -9,10 +9,10 @@ import { DockerHubV2ApiResponse, DockerHubV2Repository, DockerHubV2Tags } from "
 
 export async function getReposForNamespace(context: IActionContext, namespace: string, nextUrl?: string | null): Promise<DockerHubV2ApiResponse<DockerHubV2Repository>> {
     const url = nextUrl || `https://hub.docker.com/v2/repositories/${namespace}`;
-    return <DockerHubV2ApiResponse<DockerHubV2Repository>>(await sendRequestWithTimeout(context, { url, method: 'GET' }, 5000, undefined)).parsedBody;
+    return <DockerHubV2ApiResponse<DockerHubV2Repository>>JSON.parse((await sendRequestWithTimeout(context, { url, method: 'GET' }, 5000, undefined)).bodyAsText as string);
```

Comment:
When I changed the Azure REST pipeline stuff, it changed the response body a little bit, so that is to just to account for that.